pkg/runtime/types.go

```diff
@@ -130,6 +132,39 @@ type Unstructured struct {
 	Object map[string]interface{}
 }
 
+// UnstructuredFrom creates an Unstructured object from the provided
+// object. It is up to the caller to make sure the object has a valid
+// group, version, and kind before calling this.
+func UnstructuredFrom(obj Object) (*Unstructured, error) {
+	var buf bytes.Buffer
+	u := new(Unstructured)
+	err := UnstructuredJSONScheme.EncodeToStream(obj, &buf)
+	if err != nil {
+		return nil, err
+	}
+
+	_, _, err = UnstructuredJSONScheme.Decode(buf.Bytes(), nil, u)
+	if err != nil {
+		return nil, err
+	}
+	return u, nil
+}
+
+// PopulateObject translates the unstructured object to the object
+// provided. No conversions or defaulting is done. It is up to the
+// caller to make sure the schemas match and the translation makes
+// sense.
+func (u *Unstructured) PopulateObject(obj Object) error {
+	var buf bytes.Buffer
+	err := UnstructuredJSONScheme.EncodeToStream(u, &buf)
+	if err != nil {
+		return err
+	}
+
+	_, _, err = UnstructuredJSONScheme.Decode(buf.Bytes(), nil, obj)
```

Comment:
@deads2k I'm not sure what exactly what you mean by "default json serializations", but this scheme will fall back to `json.Unmarshal` on decode and `json.Marshal` on encode.